### ISSUE TITLE
:question: Zip Code 2020 Census Update

### ISSUE THREAD

**Issue by sethgoodluck** [crhallberg/zips#4]
Will you be updating this when the 2020 census comes out?

**Comment by crhallberg** [comment]
If I can find a good source for the data, yes! I used [GeoNames](http://www.geonames.org/) as my source so I am hopeful they will update their data. I'll work on getting my data generation code up here so we can be sure!

**Comment by teebu** [comment]
Any update on this? Zips getting a little stale.

**Comment by crhallberg** [comment]
I agree, @teebu. I've been wondering if this little library should be modernized or archived. Some of its approaches are a bit naive compared to what I've learned since the last update.

In the meantime, it does look like GeoNames has updated their data, so I can at least start with a data refresh!

**Comment by crhallberg** [comment]
I hope this helps! ~~https://github.com/crhallberg/zips/commits/master~~

EDIT: 831d60adb4b4d1fce0cea8c3367ebe0a9220715d

**Comment by teebu** [comment]
That links goes to 404. 

Thanks for the update, can you push to npmjs?

**Comment by crhallberg** [comment]
Had to call it `1.1.3` bc npm enforces semantic versioning, but 1.1.2.2033-02-03 is up on NPM: https://www.npmjs.com/package/zips/v/1.1.3.
